cmd/exec/main.go

```diff
@@ -71,14 +68,13 @@ func main() {
 
 	execCmd.Flags().StringVar(&inputType, "input-type", "application/json", "Content Type of input data")
 
-	eventCmd.PersistentFlags().StringVar(&Source, "source", "", "Context in which event happen")
-	eventCmd.PersistentFlags().StringVar(&Type, "type", "", "Type of event")
-	eventCmd.PersistentFlags().StringVar(&Id, "id", "", "Event id ")
-	eventCmd.PersistentFlags().StringVar(&Specversion, "specversion", "", "The version of the CloudEvents specification which the event uses")
+	sendEventCmd.Flags().StringVar(&Source, "source", "", "Context in which event happen")
```

Comment:
This was a mistake from a previous PR. Check the difference between `PersistentFlags` and `Flags` in the cobra docs to understand why.